extension-readme.md

```diff
@@ -1,182 +1,242 @@
 # <a id="top"></a>AWS Toolkit for Visual Studio Code
 
-The AWS Toolkit for Visual Studio Code is an extension that enables you to interact with [Amazon Web Services (AWS)](https://aws.amazon.com/what-is-aws/) from within Visual Studio Code.
+The *AWS Toolkit for Visual Studio Code* is an extension that enables you to interact with [Amazon Web Services (AWS)](https://aws.amazon.com/what-is-aws/) from within the Visual Studio Code editor.
```

Comment:
Do we want to emphasize that this is for Serverless development at this time?

Reply:
I don't really have an inclination one way or the other. Could say something like "...enables you to interact with **certain services** of Amazon Web Services (AWS) from within..." to keep it maintainable but also give them a heads-up that they need keep reading to find out what those services are.

Reply:
(I have added it to the pull request. See the Files Changed tab.)